components/sections/About.tsx

```diff
@@ -2,7 +2,7 @@
 
 const AboutSection = () => {
   return (
-    <section className='mt-10' id='aboutMe'>
+    <section data-cy='about' className='mt-10' id='aboutMe'>
```

Comment:
It should be consistent follow either `className then data` / `data then className`

Reply:
going for `data then className`